bundles/org.openhab.binding.roku/src/main/java/org/openhab/binding/roku/internal/handler/RokuHandler.java

```diff
@@ -134,6 +140,22 @@ private void refreshPlayerState() {
         synchronized (sequenceLock) {
             String activeAppId = ROKU_HOME_ID;
             try {
+                if (thingTypeUID.equals(THING_TYPE_ROKU_TV)) {
+                    try {
+                        deviceInfo = communicator.getDeviceInfo();
+                        String powerMode = deviceInfo.getPowerMode();
+                        updateState(POWER_STATE, new StringType(powerMode));
+                        if (POWERON.equalsIgnoreCase(powerMode)) {
+                            updateState(POWER, OnOffType.ON);
+                        } else {
+                            updateState(POWER, OnOffType.OFF);
```

Comment:
Could be considered:
```suggestion
                        updateState(POWER, OnOffType.from(POWERON.equalsIgnoreCase(powerMode)));
```